### DISCUSSION TITLE
Floodcam lights stay on

### DISCUSSION THREAD

**Discussion by lottomotto** [dgreif/ring#1296] [General]
Hello,

I did use ring homebridge plugin for a while.. now I switched to Scrypted because of the HKSC. It all seems to work, But I have a problem.

My floodcams turn on their lights when a motion is detected, but they don't turn off when Scrypted is on... (in Homebridge it worked normally, actually the setting comes from the ring app)

What can I do?

THanks a lot

**Comment by dgreif** [comment]
You are probably better off asking in the scrypted discord server or repo. The Scrypted Ring plugin is not maintained in this repo.
